.spelling

```diff
@@ -666,6 +666,7 @@ README.md
 certificate.md
 pageinfo
 md#renew
+upgrade.md
```

Comment:
Is this a mistake? I can't see any use of the term `upgrade.md` in this PR.
And if it is used, it can be put between backticks so that the spell checker ignores it.

Reply:
This is necessary for this page: https://github.com/cert-manager/website/pull/1317/files#diff-997f04fe6d420c950b5b0adc9a59cb1c9d229ae6653ec2eb821232faed17fc20R33

Reply:
I think it's just because the link is split across two lines. This fixes it:

```diff
diff --git a/.spelling b/.spelling
index 3bd4c0ab..6a2e900f 100644
--- a/.spelling
+++ b/.spelling
@@ -666,7 +666,6 @@ README.md
 certificate.md
 pageinfo
 md#renew
-upgrade.md

 # END TEMPORARY

diff --git a/content/docs/releases/upgrading/upgrading-0.10-0.11.md b/content/docs/releases/upgrading/upgrading-0.10-0.11.md
index 913acf51..183d9fc6 100644
--- a/content/docs/releases/upgrading/upgrading-0.10-0.11.md
+++ b/content/docs/releases/upgrading/upgrading-0.10-0.11.md
@@ -29,8 +29,8 @@ This upgrade should be performed in a few steps:
 4. Update the `apiVersion` on all your backed up resources from
    `certmanager.k8s.io/v1alpha1` to `cert-manager.io/v1alpha2`.

-5. Re-install cert-manager from scratch according to the [installation
-   guide](../../installation/upgrade.md).
+5. Re-install cert-manager from scratch according to the
+   [installation guide](../../installation/upgrade.md).

 You must be sure to properly **backup**, **uninstall**, **re-install** and
 **restore** your installation in order to ensure the upgrade is successful.
diff --git a/content/docs/releases/upgrading/upgrading-0.16-1.0.md b/content/docs/releases/upgrading/upgrading-0.16-1.0.md
index fb104342..54fd19eb 100644
--- a/content/docs/releases/upgrading/upgrading-0.16-1.0.md
+++ b/content/docs/releases/upgrading/upgrading-0.16-1.0.md
@@ -58,8 +58,8 @@ This upgrade MUST be performed in the following sequence of steps:
 4. Ensure the old cert-manager CRD resources have also been deleted: `kubectl get crd | grep cert-manager.io`


-5. Re-install cert-manager `v1.0` from scratch according to the [installation
-   guide](../../installation/upgrade.md).
+5. Re-install cert-manager `v1.0` from scratch according to the
+   [installation guide](../../installation/upgrade.md).

 6. Apply the backed up resources again.
```

Reply:
Fixed in https://github.com/cert-manager/website/pull/1317/commits/abd0be898dbdbcdcefe5d8225b4c540be3df1abc